.github/workflows/test.yml

```
@@ -24,7 +24,15 @@ jobs:
           # TODO: add jruby and truffleruby
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-${{ matrix.ruby }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
```

```suggestion
          restore-keys: ${{ runner.os }}-gems-
```
Why did you use the `|` version?

---

oh, no real reason. Thanks for removing it!